dashboard/app/views/home/_homepage.html.haml

```diff
@@ -27,19 +27,21 @@
 #homepage-container
 
 #admin-extras
-  -if current_user.admin?
-    %a{href: CDO.code_org_url('/ops-dashboard#/districts'), style: 'text-decoration: none;'}
-      %h2
-        Manage Districts
-  -if current_user.admin? || has_permission?('facilitator')
-    %a{href: CDO.code_org_url('/ops-dashboard#/workshops'), style: 'text-decoration: none;'}
-      %h2
+  -if current_user.admin? || current_user.permission?(UserPermission::CREATE_PROFESSIONAL_DEVELOPMENT_WORKSHOP) || current_user.permission?(UserPermission::WORKSHOP_ADMIN) || current_user.permission?(UserPermission::DISTRICT_CONTACT)
```

Comment:
Remind @ashercodeorg to confirm that this page hits the DB for `user_permissions` exactly once.